src/test/scala/org/camunda/feel/impl/builtin/BuiltinListFunctionsTest.scala

```diff
@@ -570,4 +570,17 @@ class BuiltinListFunctionsTest
     evaluateExpression(" is empty([1,2,3]) ") should returnResult(false)
     evaluateExpression(" is empty(list: [1]) ") should returnResult(false)
   }
+
+  "A partition() function" should "return list partitioned by _" in {
+    evaluateExpression(" partition([1,2,3,4,5], 2) ") should returnResult(
+      List(List(1, 2), List(3, 4), List(5))
+    )
```

Comment:
:wrench: We should add another case that invokes the function with named parameters.

```
evaluateExpression(" partition(list: [1,2,3,4,5], size: 2) ")
```

Refer to the [guidelines](https://github.com/camunda/feel-scala/blob/main/CONTRIBUTING.md#adding-a-new-built-in-function).